commands/orchestrationcommands/stackcommands/common.go

```diff
@@ -0,0 +1,166 @@
+package stackcommands
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/rackspace/rack/internal/github.com/fatih/structs"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+)
+
+func stackList(client *gophercloud.ServiceClient) ([]map[string]interface{}, error) {
```

Comment:
Let's put this in `list.go`. (I know several commands used to use it, but I think just `list` does now.)